src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts

```diff
@@ -233,7 +233,7 @@ describe('CampaignMonitorService', () => {
 
 		it('should log and throw error with invalid template, returns false', async () => {
 			const materialRequestEmailInfo = mockMaterialRequestEmailInfo;
-			materialRequestEmailInfo.template = Template.VISIT_DENIED;
+			materialRequestEmailInfo.template = Template.VISIT_DENIED; // Denied template is null and triggers the error
```

Comment:
Is this comment still needed. I though you were able to mock the env vars correctly by wrapping them in a function